app/components/primer/beta/auto_complete.rb

```diff
@@ -95,11 +107,23 @@ class AutoComplete < Primer::Component
       #   <% end %>
       #
       # @example With Icon
-      #   <%= render(Primer::Beta::AutoComplete.new("aria-label": "Fruits", src: "/auto_complete", list_id: "fruits-popup-5", input_id: "fruits-input-5", position: :relative)) do |c| %>
+      #   <%= render(Primer::Beta::AutoComplete.new("aria-label": "Fruits", src: "/auto_complete", list_id: "fruits-popup-6", input_id: "fruits-input-6", position: :relative)) do |c| %>
       #     <% c.input(type: :text) %>
       #     <% c.icon(icon: :search) %>
       #   <% end %>
       #
+      # @example With Screen Reader Feedback Element - TODO IN COMMENTS
+      #   <%= render(Primer::Beta::AutoComplete.new(src: "/auto_complete", input_id: "fruits-input-7", list_id: "fruits-popup-7", position: :relative)) do |c| %>
+      #     <% c.label(classes:"").with_content("Fruits") %>
+      #     <% c.input(type: :text) %>
+      #   <% end %>
+      #
+      # @example With Custom Clear Button - TODO IN COMMENTS
+      #   <%= render(Primer::Beta::AutoComplete.new(src: "/auto_complete", input_id: "fruits-input-8", list_id: "fruits-popup-8", position: :relative)) do |c| %>
+      #     <% c.label(classes:"").with_content("Fruits") %>
+      #     <% c.input(type: :text) %>
```

Comment:
We need to add how a PVC user would include the following:
`<button id="fruits-input-8-clear">x</button>`